Sources/KSCrashRecording/KSCrashReport.c

```diff
@@ -1440,6 +1441,18 @@ static void writeError(const KSCrashReportWriter* const writer,
                 writer->addStringElement(writer, KSCrashField_Type, KSCrashExcType_Signal);
                 break;
 
+            case KSCrashMonitorTypeMemoryTermination:
+            {
+                writer->addStringElement(writer, KSCrashField_Type, KSCrashExcType_MemoryTermination);
+                writer->beginObject(writer, KSCrashField_MemoryTermination);
+                {
+                    writer->addStringElement(writer, KSCrashField_MemoryPressure, crash->AppMemory.pressure);
+                    writer->addStringElement(writer, KSCrashField_MemoryLevel, crash->AppMemory.level);
```

Comment:
`writeAppMemoryInfo` is the part that gets written to every report regardless if it's memory related or not.
The other place is in the storage for the `memory_termination` type. Since they are the two pieces of info required to know if we have a memory issue, they go there as well.